projects/libvpx/vpx_dec_fuzzer.cc

```diff
@@ -77,7 +80,7 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   const unsigned int threads = 1;
 #elif defined(DECODE_MODE_threaded)
   // Set thread count in the range [2, 64].
-  const unsigned int threads = std::max((header[0] & 0x3f) + 1, 2);
+  const unsigned int threads = std::max((data[IVF_FILE_HDR_SZ] & 0x3f) + 1, 2);
```

Comment:
in my testing using header[0] gave full range coverage quickly when fuzzing, the same with libaom

Reply:
What I saw was when starting without a corpus, entire range is covered, but when starting with a corpus of *.ivf files, I was seeing number of cores being set to 5 most of the times.

Reply:
Since this is still data that isn't sent to the decoder I think it's all right, that was the preference in earlier reviews for libaom with examples given [1][2]. We should double check libaom's behavior as I thought I had run it against a corpus, but it's possible I lost track and ran it with random input to determine that coverage.

[1] https://github.com/ImageMagick/ImageMagick/blob/master/Magick++/fuzz/rotate_fuzzer.cc#L9-L16
[2] https://cs.chromium.org/chromium/src/base/json/json_reader_fuzzer.cc?type=cs&q=json_reader_fuzzer&sq=package:chromium&g=0&l=20

Reply:
Sure. will check av1 and update this code accordingly.

Reply:
I added a print for number of threads and following are the observations.
In VP9, using a corpus of *.ivf files, in 5060 calls to the decoder, number of threads was set to 5 for 5000 times when using the first byte. 
With the new method of using 33rd byte, threads takes many different values.

In AV1, more than 300 out of 500 calls, number of threads was 5.
Please let me know, if you would like me to revert this change and initialize number of cores based on the first byte.

Reply:
Thanks for taking a look, if we're getting better coverage with this byte I think it's fine to keep -- originally I had been hashing the whole header, but the preference by reviewers was to use a single value not forwarded to the decoder.